### DISCUSSION TITLE
I want to start making a good website through the blacksheep

### DISCUSSION THREAD

**Discussion by Legopapurida** [Neoteroi/BlackSheep#149] [Q&A]
I don't know that whether this will be a good target or not but I want to do this just for fun and test.
coz I really loved blacksheep.

Why your project couldn't go ahead over the fastapi ?
I think blacksheep has potential more than fastapi

**Comment by myusko** [comment]
Hi @Legopapurida 

Thank you for such kind words, I recommend you to check a demo project created by @RobertoPrevato https://github.com/Neoteroi/Venezia/tree/dev

Thanks,
Mykhailo
